Modules/Core/Common/src/itkThreadPool.cxx

```diff
@@ -154,6 +158,23 @@ ThreadPool::~ThreadPool()
   }
 }
 
+void
+ThreadPool::atfork_prepare()
+{
+  printf("In ThreadPool::atfork_prepare");
```

Comment:
Debugging comments should be removed.

Reply:
Once this PR gets to a working stage, I will remove them. Currently, they don't get printed.